workflow/Snakefile

```diff
@@ -40,20 +40,34 @@ rule all:
 			os.path.join(RESULTS_DIR, "{biosample}", "encode_e2g_predictions.tsv.gz"), biosample=BIOSAMPLES
 		),
 		# Wei-Lin: Generate the following output file
-		# expand(
-		# 	os.path.join(RESULTS_DIR, "{biosample}", "Predictions", "EnhancerPredictionsAllPutative_Kendall.tsv.gz"), biosample=BIOSAMPLES
-		# )
+		expand(
+			os.path.join(RESULTS_DIR, "{biosample}", "Predictions", "EnhancerPredictionsAllPutative_ARC.tsv.gz"), biosample=BIOSAMPLES
+		)
 
 
 ### Wei-Lin: fill out
 rule kendall_correlation:
 	input:
-		abc_predictions = os.path.join(RESULTS_DIR, "{biosample}", "Predictions", "EnhancerPredictionsAllPutative.tsv.gz"),
-		# Whatever else you need ...
-		# e.g candidate_regions = os.path.join(RESULTS_DIR, "{biosample}", "Peaks", "macs2_peaks.narrowPeak.sorted.candidateRegions.bed"),
+		kendall_candidate_e2g_pairs = config["kendall_candidate_e2g_pairs"],
+		sc_atac_matix = config["sc_atac_matix"],
+		sc_rna_matix = config["sc_rna_matix"],
```

Comment:
How do these get produced? Are they derived from a separate file? e.g multiome file? 

We'll probably want to also include the steps to produce this data in the pipeline

Reply:
So based on today's discussion, let's see if we can try to utilize ABC's peaks (contingent on if we can do Kendall across a single cell cluster instead of all of them)

Feel free to use the fragment file as input for now (I'll add a rule later so that we can convert the fragment file to tagAlign, which ABC will use).